reporter/src/web-app/js/src/index.css

```diff
@@ -5,14 +5,41 @@ body {
     font-family: sans-serif;
```

Comment:
Believe this should not go into index.css but App.css as it's App related.

Reply:
that should go to another PR I assume as soon as that is not a change done within current PR

Reply:
index.css - that is top level css that should handle html/body/and other stuff outside App
app - that is wrapper for the App component, and the App.css should contain only stuff related to App component and it's children